echopype/utils/prov.py

```diff
@@ -5,7 +5,7 @@
 from _echopype_version import version as ECHOPYPE_VERSION
 from typing_extensions import Literal
 
-ProcessType = Literal["conversion", "processing"]
+ProcessType = Literal["conversion", "combination", "processing"]
```

Comment:
If we decide to not change the name in this PR (see my previous comment), then we should also remove this.

Reply:
See my comment above